front/src/index.js

```diff
@@ -2,9 +2,49 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import Employees from './Employees/Employees'
+import Events from './Events/Events'
+import Incomes from './Incomes/Incomes'
+import Outcomes from './Outcomes/Outcomes'
+import Bar from './LinksUpperBar/Bar'
+// import Notfound from './NotFound/notfound'
+import {Route, Link, BrowserRouter as Router} from 'react-router-dom'
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+
+const routing = (
+    <Router>
+        <div>
+            <ul>
+                <li>
+                    <Link to="/">Home</Link>
+                </li>
+                <li>
+                    <Link to="/employees">Employees</Link>
+                </li>
+                <li>
+                    <Link to="/events">Events</Link>
+                </li>
+                <li>
+                    <Link to="/incomes">Incomes</Link>
+                </li>
+                <li>
+                    <Link to="/outcomes">Outcomes</Link>
+                </li>
+            </ul>
+            {/*<Switch>*/}
+            <Route path="/" component={App}/>
+            <Route path="/employees" component={Employees}/>
```

Comment:
Do we have an example of detail view, e.g details of single employee? One detail path would make you more familiar with specifics of passing params in urls in react-router